js/private/js_library.bzl

```diff
@@ -214,12 +214,14 @@ def _js_library_impl(ctx):
 
     return [
         js_info(
+            target = ctx.label,
+            bin_dir = ctx.bin_dir,
```

Comment:
Can't `ctx.bin_dir` be used without having to copy it here? The only place I see this used it does:
```
ctx.attr.src[JsInfo].bin_dir.path
```
Would `ctx.bin_dir` not be correct there?

Reply:
It's not clear if it would be the same bin_dir in the rule context with transitions. To be safe the bin dir of the target that created the provider should always be correct

Reply:
In theory, the two bin_dirs _should_ always be the same since if the npm_package_store is transitioned so should all of js_library targets that it depends on. Lets go with `ctx.bin_dir` for now for simplicity and if it ends up being an issue then it will be bug fix in the future.